packages/functional-tests/src/tests/sound-test.ts

```diff
@@ -0,0 +1,105 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as MRESDK from '@microsoft/mixed-reality-extension-sdk';
+import { Actor } from '@microsoft/mixed-reality-extension-sdk';
+import App from '../app';
+import delay from '../utils/delay';
+import destroyActors from '../utils/destroyActors';
+import Test from './test';
+
+export default class SoundTest extends Test {
+
+    constructor(app: App, private baseUrl: string) {
+        super(app);
+    }
+
+    public async run(): Promise<boolean> {
+        let success = true;
+
+        success = success && await this.runSoundTest();
+
+        return success;
+    }
```

Comment:
I see you using this pattern a lot in functional tests, is this code doing anything at all? Seems like all it does is return the value from the actual test runner.

Reply:
dunno, I copied and pasted. If you change for other tests then I will adapt for this

---
In reply to: [258149899](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/204#discussion_r258149899) [](ancestors = 258149899)